Content.Server/Atmos/Piping/Trinary/EntitySystems/GasFilterSystem.cs

```diff
@@ -211,5 +212,20 @@ private void OnFilterAnalyzed(EntityUid uid, GasFilterComponent component, GasAn
 
             args.DeviceFlipped = inlet != null && filterNode != null && inlet.CurrentPipeDirection.ToDirection() == filterNode.CurrentPipeDirection.ToDirection().GetClockwise90Degrees();
         }
+
+
+        // Frontier - Start: Enable filters at roundstart
+        private void OnMapInit(EntityUid uid, GasFilterComponent filter, MapInitEvent args) // Frontier - Init on map
```

Comment:
Remove extra blank line
```suggestion

        // Frontier - Start: Enable filters at roundstart
        private void OnMapInit(EntityUid uid, GasFilterComponent filter, MapInitEvent args) // Frontier - Init on map
```